operations/experimental/use-logcache-syslog-ingress.yml

```diff
@@ -21,7 +21,7 @@
     type: certificate
     options:
       ca: loggregator_ca
-      common_name: log-cache-syslog
```

Comment:
Is this a required changed for the removals to work? Or could this be a separate change.

Reply:
It can be a separate change but syslog ingress into log-cache needs to work in order to disable the v2 firehose and it won't without this change so they are semi-related. We can pull it into a different PR if you would prefer

Reply:
Thanks for the info. No I think this change is okay to be pulled in this PR since it's experimental.